### PR TITLE
Switch port numbers for in-coordinator/extracted-drop-wizard cases

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

Fixes port number of in-coordinator REST API for SGv2 to be same as in SGv1 (8082); switches now-obsolete DropWizard-based "new" REST API to 8083 until it can be safely removed. And modifies test base class to use that port number.

**Which issue(s) this PR fixes**:

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: sgv2-restapi/src/main/resources/config.yaml [hunk 1]
```diff
@@ -3,9 +3,9 @@ server:
   applicationContextPath: /
   connector:
     type: http
-    # 11-Feb-2022, tatu: For Stargate V1, REST+Docs were both on port 8082, but for Stargate V2
-    #    will separate the two and the "new" REST API inherits 8082
-    port: 8082
+    # 11-Feb-2022, tatu: This implementation will be removed soon, but until then
+    #    has to expose port OTHER than 8082 for ITs to work
+    port: 8083
```
Comment (ivansenic) [inline sgv2-restapi/src/main/resources/config.yaml:8]: Why is this change needed? What integration tests would not work? I think this is not needed imo..

Comment (ivansenic) [inline sgv2-restapi/src/main/resources/config.yaml:8]: Why is this change needed? What integration tests would not work? I think this is not needed imo..

**Review by ivansenic — COMMENTED** [review]
This is very confusing to me.. Maybe a quick sync..

**Comment by tatu-at-datastax** [comment]
@ivansenic yes, meeting now.

**Comment by tatu-at-datastax** [comment]
@ivansenic Are we good now? Hoping to just merge this and proceed with eliminating parts as discussed.

File: sgv2-restapi/src/main/resources/config.yaml (hunk 1 above)
Comment (tatu-at-datastax) [reply sgv2-restapi/src/main/resources/config.yaml:8]: (discussed in meeting -- is due to DW-based impl not yet removed due to IT dependencies to it -- will be removed once ITs deleted/converted)

**Review by jeffreyscarpenter — APPROVED** [review]

**Comment by ivansenic** [comment]
Go for it..
